{"uid":"9c8bb8362a00b659","name":"test_discourse","fullName":"tests.integration.new_relations.test_new_relations#test_discourse","historyId":"48afa2f8009ffdd3214ff6ed4338f118","time":{"start":1728441041041,"stop":1728441782824,"duration":741783},"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ndiscourse-k8s/0 [idle] waiting: Waiting for redis relation to initialize","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/new_relations/test_new_relations.py\", line 637, in test_discourse\n    await ops_test.model.wait_for_idle(apps=[DISCOURSE_APP_NAME], status=\"active\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\ndiscourse-k8s/0 [idle] waiting: Waiting for redis relation to initialize","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":1,"retriesStatusChange":true,"beforeStages":[{"name":"event_loop","time":{"start":1728439900904,"stop":1728439900905,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"tmp_path_factory","time":{"start":1728439900905,"stop":1728439900905,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test","time":{"start":1728439900905,"stop":1728439901190,"duration":285},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"abort_on_fail","time":{"start":1728441041040,"stop":1728441041040,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"testStage":{"status":"broken","statusMessage":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ndiscourse-k8s/0 [idle] waiting: Waiting for redis relation to initialize","statusTrace":"Traceback (most recent call last):\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 341, in from_call\n    result: TResult | None = func()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 242, in <lambda>\n    lambda: runtest_hook(item=item, **kwds), when=when, reraise=reraise\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 92, in pytest_runtest_call\n    yield from thread_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/threadexception.py\", line 68, in thread_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 95, in pytest_runtest_call\n    yield from unraisable_exception_runtest_hook()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/unraisableexception.py\", line 70, in unraisable_exception_runtest_hook\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 846, in pytest_runtest_call\n    yield from self._runtest_for(item, \"call\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/logging.py\", line 829, in _runtest_for\n    yield\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/capture.py\", line 880, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/skipping.py\", line 257, in pytest_runtest_call\n    return (yield)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/runner.py\", line 174, in pytest_runtest_call\n    item.runtest()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 1627, in runtest\n    self.ihook.pytest_pyfunc_call(pyfuncitem=self)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_hooks.py\", line 513, in __call__\n    return self._hookexec(self.name, self._hookimpls.copy(), kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_manager.py\", line 120, in _hookexec\n    return self._inner_hookexec(hook_name, methods, kwargs, firstresult)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 182, in _multicall\n    return outcome.get_result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_result.py\", line 100, in get_result\n    raise exc.with_traceback(exc.__traceback__)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pluggy/_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/_pytest/python.py\", line 159, in pytest_pyfunc_call\n    result = testfunction(**testargs)\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/pytest_asyncio/plugin.py\", line 529, in inner\n    _loop.run_until_complete(task)\n  File \"/usr/lib/python3.10/asyncio/base_events.py\", line 649, in run_until_complete\n    return future.result()\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/tests/integration/new_relations/test_new_relations.py\", line 637, in test_discourse\n    await ops_test.model.wait_for_idle(apps=[DISCOURSE_APP_NAME], status=\"active\")\n  File \"/home/ubuntu/actions-runner/_work/test-runners-2-is-x64-postgresql-k8s-operator/test-runners-2-is-x64-postgresql-k8s-operator/.tox/integration/lib/python3.10/site-packages/juju/model.py\", line 2970, in wait_for_idle\n    raise jasyncio.TimeoutError(\"Timed out waiting for model:\\n\" + busy)\nasyncio.exceptions.TimeoutError: Timed out waiting for model:\ndiscourse-k8s/0 [idle] waiting: Waiting for redis relation to initialize","steps":[],"attachments":[{"uid":"b7f7c05d9824ae61","name":"log","source":"b7f7c05d9824ae61.txt","type":"text/plain","size":4806}],"parameters":[],"stepsCount":0,"shouldDisplayMessage":true,"attachmentsCount":1,"hasContent":true,"attachmentStep":false},"afterStages":[{"name":"abort_on_fail::0","time":{"start":1728441782828,"stop":1728441782828,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"ops_test::finalizer","time":{"start":1728442032750,"stop":1728442033267,"duration":517},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::2","time":{"start":1728442033268,"stop":1728442033268,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_close_event_loop","time":{"start":1728442033268,"stop":1728442033268,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false},{"name":"event_loop::_provide_clean_event_loop","time":{"start":1728442033268,"stop":1728442033269,"duration":1},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"shouldDisplayMessage":false,"attachmentsCount":0,"hasContent":false,"attachmentStep":false}],"labels":[{"name":"tag","value":"@pytest.mark.skipif(False, reason='Requires amd64 architecture')"},{"name":"tag","value":"group(1)"},{"name":"tag","value":"asyncio"},{"name":"parentSuite","value":"tests.integration.new_relations"},{"name":"suite","value":"test_new_relations"},{"name":"host","value":"github-runner"},{"name":"thread","value":"21955-MainThread"},{"name":"framework","value":"pytest"},{"name":"language","value":"cpython3"},{"name":"package","value":"tests.integration.new_relations.test_new_relations"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[{"uid":"c887f07b7a961747","status":"unknown","time":{}}],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":5,"skipped":72,"passed":35,"unknown":10,"total":122},"items":[{"uid":"9b84de05cd1e5399","reportUrl":"../200//#testresult/9b84de05cd1e5399","status":"passed","time":{"start":1728353070585,"stop":1728353528899,"duration":458314}},{"uid":"e4b6bd1264e9ea03","reportUrl":"../199//#testresult/e4b6bd1264e9ea03","status":"passed","time":{"start":1728268531852,"stop":1728268925136,"duration":393284}},{"uid":"d715f7c0b856661e","reportUrl":"../198//#testresult/d715f7c0b856661e","status":"passed","time":{"start":1728181258235,"stop":1728181616606,"duration":358371}},{"uid":"f379215b21159650","reportUrl":"../197//#testresult/f379215b21159650","status":"passed","time":{"start":1728093648165,"stop":1728094014691,"duration":366526}},{"uid":"5851172c4d2c20fd","reportUrl":"../196//#testresult/5851172c4d2c20fd","status":"passed","time":{"start":1728007280235,"stop":1728007689966,"duration":409731}},{"uid":"b2e68b102ceba7f7","reportUrl":"../194//#testresult/b2e68b102ceba7f7","status":"passed","time":{"start":1727834461460,"stop":1727834843771,"duration":382311}},{"uid":"921a6793c607ec04","reportUrl":"../193//#testresult/921a6793c607ec04","status":"broken","statusDetails":"asyncio.exceptions.TimeoutError: Timed out waiting for model:\ndiscourse-k8s/0 [idle] waiting: Waiting for redis relation","time":{"start":1727747862103,"stop":1727748562527,"duration":700424}},{"uid":"b539ef223cbe9455","reportUrl":"../192//#testresult/b539ef223cbe9455","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727660107006,"stop":1727660107006,"duration":0}},{"uid":"c76c92cbab3755c9","reportUrl":"../191//#testresult/c76c92cbab3755c9","status":"passed","time":{"start":1727576610940,"stop":1727576965241,"duration":354301}},{"uid":"6c397fd4cffb1d13","reportUrl":"../190//#testresult/6c397fd4cffb1d13","status":"passed","time":{"start":1727488613992,"stop":1727488949102,"duration":335110}},{"uid":"415d758c571d5e5d","reportUrl":"../189//#testresult/415d758c571d5e5d","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727403826288,"stop":1727403826288,"duration":0}},{"uid":"667930e554d04034","reportUrl":"../188//#testresult/667930e554d04034","status":"passed","time":{"start":1727318884655,"stop":1727319209124,"duration":324469}},{"uid":"f6277777af9f95c9","reportUrl":"../186//#testresult/f6277777af9f95c9","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727229502627,"stop":1727229502627,"duration":0}},{"uid":"802251fd39b76ef","reportUrl":"../185//#testresult/802251fd39b76ef","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727142800894,"stop":1727142800894,"duration":0}},{"uid":"a237766159c92713","reportUrl":"../184//#testresult/a237766159c92713","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1727055391676,"stop":1727055391676,"duration":0}},{"uid":"270af413323c473f","reportUrl":"../183//#testresult/270af413323c473f","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726969152695,"stop":1726969152695,"duration":0}},{"uid":"55bf3fafa79bc8da","reportUrl":"../182//#testresult/55bf3fafa79bc8da","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726882575415,"stop":1726882575415,"duration":0}},{"uid":"f142b6dfda1864b9","reportUrl":"../181//#testresult/f142b6dfda1864b9","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726796112089,"stop":1726796112089,"duration":0}},{"uid":"abb2937b93dc08ef","reportUrl":"../180//#testresult/abb2937b93dc08ef","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726709843058,"stop":1726709843058,"duration":0}},{"uid":"91d511db06139239","reportUrl":"../179//#testresult/91d511db06139239","status":"skipped","statusDetails":"XFAIL reason: aborted\n\n_pytest.outcomes.XFailed: aborted","time":{"start":1726623321224,"stop":1726623321224,"duration":0}}]},"tags":["asyncio","@pytest.mark.skipif(False, reason='Requires amd64 architecture')","group(1)"]},"source":"9c8bb8362a00b659.json","parameterValues":[]}